aws-xray-propagator/src/main/java/io/opentelemetry/contrib/awsxray/propagator/AwsXrayPropagator.java

```diff
@@ -241,9 +199,6 @@ private static <C> Context getContextFromHeader(
     if (spanContext.isValid()) {
       context = context.with(Span.wrap(spanContext));
     }
-    if (baggage != null) {
-      context = context.with(baggage.build());
```

Comment:
Can we use a new key "XRAY_HEADER_ADDITIONAL_FIELDS"
```
context = context.with(XRAY_HEADER_ADDITIONAL_FIELDS ,baggage.build());
```
FYI https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1178#issuecomment-2050711021